Newer Kernels have more reasonable defaults for dirty_ratio, use the kernels
defaults by default also for laptop-mode.

Index: b/etc/laptop-mode/laptop-mode.conf
===================================================================
--- a/etc/laptop-mode/laptop-mode.conf
+++ b/etc/laptop-mode/laptop-mode.conf
@@ -293,11 +293,11 @@ CONTROL_MOUNT_OPTIONS=1
 #
 # Dirty synchronous ratio.  At this percentage of dirty pages the process
 # which calls write() does its own writeback.
 #
 LM_DIRTY_RATIO=60
-NOLM_DIRTY_RATIO=40
+NOLM_DIRTY_RATIO=20
 
 
 #
 # Allowed dirty background ratio, in percent.  Once DIRTY_RATIO has been
 # exceeded, the kernel will wake pdflush which will then reduce the amount
Index: b/usr/sbin/laptop_mode
===================================================================
--- a/usr/sbin/laptop_mode
+++ b/usr/sbin/laptop_mode
@@ -74,7 +74,7 @@
 LM_DIRTY_RATIO=60
 LM_DIRTY_BACKGROUND_RATIO=1
 NOLM_DIRTY_BACKGROUND_RATIO=10
-NOLM_DIRTY_RATIO=40
+NOLM_DIRTY_RATIO=20
 LM_SECONDS_BEFORE_SYNC=2
 BATT_CPU_MAXFREQ=medium
 BATT_CPU_MINFREQ=slowest

